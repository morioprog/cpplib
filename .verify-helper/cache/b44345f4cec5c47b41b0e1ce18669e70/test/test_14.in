10
-2 9
-9 4
-1 5
-3 4
8 1
-2 3
-4 7
-1 1
-7 1
-5 4
