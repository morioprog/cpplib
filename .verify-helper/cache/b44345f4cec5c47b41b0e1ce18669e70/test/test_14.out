8 1
-1 5
-2 9
-4 7
-2 3
-3 4
-1 1
-5 4
-9 4
-7 1
